### PR TITLE
grass tiles added, movement function added, resized to fit all square…

### PR DESCRIPTION
**Pull request by JimD325**
…s on screen. TODO have buttons component change position based on whether or not the screen is taller or wider

### CODE REVIEW

**Review by ZayahPapaya — APPROVED** [review]

File: src/components/GameBoard/GameBoard.css [hunk 1]
```diff
@@ -3,14 +3,18 @@
   border: 1px solid black;
   border-bottom: none;
   border-right: none;
-  background-color: #229557;
+  /* background-color: #229557; */
```
Comment (laptopadventure) [inline src/components/GameBoard/GameBoard.css:6]: remove commented code

File: src/components/GameBoard/GameBoard.css [hunk 2]
```diff
@@ -23,18 +27,18 @@
...
-  /* transition: margin 0.5s ease-in-out; */
+  transition: margin 0.5s ease-in-out;
   margin: 1px 1px;
   
 }
 
 .raccoon.damage {
   position: absolute;
-  transition: margin 0.5s ease-in-out;
+  /* transition: margin 0.5s ease-in-out; */
```
Comment (laptopadventure) [inline src/components/GameBoard/GameBoard.css:37]: commented code

File: src/components/GameBoard/GameBoard.tsx [hunk 3]
```diff
@@ -113,23 +131,39 @@ const Board = (props: BoardProps) => {
 // TODO Conditionally render Pause/play button depending on whether or not the game is playing. 
 const Buttons = () => {
   const { setScreen } = useContext(ScreenContext)
+  const { winWidth, winHeight } = useWindowDimensions()
+  if (winWidth > winHeight) {
+    console.log("wide")
+    return (
+
```
Comment (laptopadventure) [inline src/components/GameBoard/GameBoard.tsx:138]: useless space

File: src/components/GameBoard/GameBoard.tsx [hunk 4]
```diff
@@ -113,23 +131,39 @@ const Board = (props: BoardProps) => {
...
-    </div>
-  )
 
 }
 
 const GameBoard = () => {
-  const { winWidth } = useWindowDimensions()
   return (
     <div className="gameBoard">
+
```
Comment (laptopadventure) [inline src/components/GameBoard/GameBoard.tsx:164]: remove useless space

**Review by laptopadventure — CHANGES_REQUESTED** [review]

**Pull request merged**